### PR TITLE
Bugfix/change submitbtn

### PR DESCRIPTION
**Pull request by kendra4227**
Created a new PR and increased the label font size to 18 px 
Changed the Submit button with the approved background color from Figma.

Referenced to https://trello.com/c/wbyKwyNm/648-import-parentcontact-from-components-pages-parentcontact

### CODE REVIEW

File: src/styles/less/ParentContact.less [hunk 1]
```diff
@@ -77,7 +77,7 @@
   padding: 7px 20px;
   margin-right: 50px;
   border-radius: 16px;
-  background-color: #e5e5e5;
+  background-color: #2d2a2d;
```
Comment (just-kato) [inline src/styles/less/ParentContact.less:80]: Nice work! Darker is always better. Super easy to see.

File: src/styles/less/ParentContact.less [hunk 2]
```diff
@@ -93,7 +93,7 @@
 }
 
 label {
-  font-size: 12px;
+  font-size: 18px;
```
Comment (just-kato) [inline src/styles/less/ParentContact.less:96]: Nice work

**Review by just-kato — DISMISSED** [review]

**Review by Alisa1989 — APPROVED** [review]

**Review by milklot — APPROVED** [review]
Nice work, looks good

**Review by just-kato — APPROVED** [review]

File: src/styles/less/ParentContact.less (hunk 2 above)
Comment (artofmayhem) [reply src/styles/less/ParentContact.less:96]: nice work bringing elements to design spec

**Review by artofmayhem — APPROVED** [review]
Well executed and ready for merge

**Pull request merged**